Resources/Prototypes/Entities/Structures/Machines/fax_machine.yml

```diff
@@ -0,0 +1,102 @@
+﻿- type: entity
+  parent: BaseMachinePowered
+  id: FaxMachineBase
+  name: long range fax machine
+  description: Bluespace technologies on the application of bureaucracy.
+  components:
+  - type: Sprite
+    sprite: Structures/Machines/fax_machine.rsi
+    netsync: false
+    layers:
+    - state: icon
+      map: ["base"]
+  - type: Icon
+    sprite: Structures/Machines/fax_machine.rsi
+    state: icon
+  - type: Appearance
+  - type: Physics
+    bodyType: Static
+  - type: Fixtures
+    fixtures:
+    - shape:
+        !type:PhysShapeAabb
+        bounds: "-0.25,-0.4,0.25,0.4"
```

Comment:
the bounds seem like they should be rotated by 90 degrees?
![Content Client_MboHIC95GC](https://user-images.githubusercontent.com/60421075/204424200-18450482-f531-4fe1-a6c6-9fa62419f4e1.png)